### ISSUE TITLE
Private Modules Missing when tfsec run at root of many sub Terraform folders

### ISSUE THREAD

**Issue by medoix** [tfsec/tfsec#546]
**Describe the bug**
When running tfsec at the root of a multi project folder i am getting many Warnings about remote modules but when tfsec is run inside of the sub projects terraform folder it works not problems.

`WARNING: Failed to load module: missing module with source 'git@github.com:ORG/modules.git' -  try to 'terraform init' first`

**To Reproduce**
1. Have a repo containing many sub and sub-sub folders that contain terraform folders that contain remote modules
2. Run tfsec at the root of the folder and see many warnings related to remove modules
3. See errors
3. Run tfsec within the terraform folders of the sub and sub-sub folders and see it work without issue

**Expected behavior**
I would expect that tfsec itterates over the sub and sub-sub folders and returns findings for each file instead of throwing these warnings.

**Desktop (please complete the following information):**
 - OS: OS X and within Docker container
 - Version: v0.36.8

**Comment by owenrumney** [comment]
Hey @medoix  - we've made the active decision to not keep going through sub folders. When we hit the first folder with the checks are run in there. When a terraform init is run, the .terraform folder will have the nested module dependencies loaded and we parse them all. 

TLDR, tfsec deliberately won't keep traversing deep into a nested folder structure (not withstanding .terraform modules)
